front/lib/api/tables.ts

```diff
@@ -446,6 +446,8 @@ export async function rowsFromCsv({
             DateTime.fromSQL,
             // Google Spreadsheet date format parser.
             (text: string) => DateTime.fromFormat(text, "d-MMM-yyyy"),
+            // Full month name format parser
+            (text: string) => DateTime.fromFormat(text, "LLLL dd, yyyy"),
```

Comment:
Can you also add here please ? 🙏 https://github.com/dust-tt/dust/blob/4badfaba8d9c91d4398d6cd85c028682b8ca646c/core/src/databases/table.rs#L665-L677